<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="quitterButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>740</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>1101</width>
     <height>401</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1101</width>
     <height>0</height>
    </size>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Repère</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Fournisseur</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Paiement</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>N°facture</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Montant</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Taux TVA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Montant TVA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Commentaire</string>
    </property>
   </column>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>256</width>
     <height>190</height>
    </rect>
   </property>
   <property name="gridVisible">
    <bool>false</bool>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>10</y>
     <width>511</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Gestion Des Recettes : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="moisLabel">
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Mois&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="anneeLabel">
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Année&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidgetBoutons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>411</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="bouton">
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonEffacer">
      <property name="toolTip">
       <string extracomment="Effacer tous les champs du formulaire"/>
      </property>
      <property name="text">
       <string>Effacer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButtonModifier">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButtonValider">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="push_calculettettc">
      <property name="text">
       <string>Calculette TTC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonSuprimer">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>740</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>216</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>N°facture</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Paiement</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelTVA">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TVA :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEditMontantTVA"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditMontant"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelForniseur">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Client&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelMontantTVA">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Montant TVA :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditnfacture"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditDate"/>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBoxpayment"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelTTC">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Montant&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEditComentaire"/>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBoxTVA"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelComentaire">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Comentaire :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxFournisseur"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelDate">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Date :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdittotalmontanttva">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>762</y>
     <width>132</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelmontanttva">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>762</y>
     <width>99</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Montant TVA</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltotalttc">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>762</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Montant Total&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdimontanttotal">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>762</y>
     <width>132</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <zorder>pushButtonSuprimer</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lineEdittotalmontanttva</zorder>
  <zorder>labelmontanttva</zorder>
  <zorder>labeltotalttc</zorder>
  <zorder>lineEdimontanttotal</zorder>
  <zorder>layoutWidgetBoutons</zorder>
  <zorder>splitter</zorder>
  <zorder>quitterButton</zorder>
  <zorder>tableWidget</zorder>
  <zorder>calendarWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
